examples/05_nossr/src/pages/_layout.tsx

```diff
@@ -40,5 +40,5 @@ const getData = async () => {
 export const getConfig = async () => {
   return {
     render: 'static',
-  };
+  } as const;
```

Comment:
Yeah that also works. I chose to put it on the full object as a convention because when `staticPaths` and `render` are both defined this way requires you to write `as const` just once, but `'static' as const` would also require `staticPaths: ['foo'] as const`